src/components/WwwFrame/WwwPage.vue

```diff
@@ -90,6 +92,7 @@ export default {
 			return [
 				this.mainClass,
 				{ 'tw-bg-secondary': this.grayBackground },
+				{ 'md:tw-mt-9 tw-mt-8': this.enableBasketExperiment }
```

Comment:
This solution is causing page sizing pop-in (see below for initial page rendering). Please investigate alternatives to get the server render correct.
![image](https://github.com/user-attachments/assets/66012756-bc79-44df-b2a2-2988d1a35f00)

Reply:
This is with the experiment enabled, since the header is sticky. The sticky header is why the content is higher up until the loans load, enable experiment, and push down with margin.